js/sigin.js

```diff
@@ -0,0 +1,82 @@
+const emailInput = document.getElementById("emailInput");
+const noEmail = document.getElementById("noEmail");
+const wrongEmail = document.getElementById("wrongEmail");
+
+emailInput.addEventListener("focusout", function () {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+  if (!emailInput.value.trim()) {
+    // 이메일을 입력해주세요 에러 메세지
+    noEmail.style.display = "block";
+    wrongEmail.style.display = "none";
+    emailInput.style.borderColor = "var(--red)";
+  } else if (!emailRegex.test(emailInput.value)) {
+    // 이메일 형식 오류 에러 메세지
+    noEmail.style.display = "none";
+    wrongEmail.style.display = "block";
+    emailInput.style.borderColor = "var(--red)";
+  } else {
+    noEmail.style.display = "none";
+    wrongEmail.style.display = "none";
+    emailInput.style.borderColor = "var(--gray20)";
+  }
+});
+
+const passwordInput = document.getElementById("passwordInput");
+const noPassword = document.getElementById("noPassword");
+
+passwordInput.addEventListener("focusout", function () {
+  if (!passwordInput.value.trim()) {
+    // 비밀번호를 입력해주세요 에러 메세지
+    noPassword.style.display = "block";
+    passwordInput.style.borderColor = "var(--red)";
+  } else {
+    noPassword.style.display = "none";
+    passwordInput.style.borderColor = "var(--gray20)";
+  }
+});
+
+const checkEmail = document.getElementById("checkEmail");
+const checkPassword = document.getElementById("checkPassword");
+
+function checkAccount() {
+  const emailValue = emailInput.value;
+  const passwordValue = passwordInput.value;
+  const rightAccount = {
+    email: "test@codeit.com",
+    password: "codeit101",
+  };
+
+  if (
+    emailValue === rightAccount.email &&
+    passwordValue === rightAccount.password
+  ) {
+    alert("로그인하시겠습니까?");
```

Comment:
사용자로부터 응답을 받을 수 있는 [window.confirm](https://developer.mozilla.org/ko/docs/Web/API/Window/confirm) 함수를 사용하면 사용자가 로그인 여부를 선택할 수 있겠네요!
```suggestion
    if (!window.confirm("로그인하시겠습니까?")) {
      return;
    }
    window.location.href = "/folder.html";
    return false;
```